libknet/libknet.h

```diff
@@ -656,9 +656,9 @@ struct knet_handle_compress_cfg {
  * knet_handle_compress_cfg -
  *            pointer to a knet_handle_compress_cfg structure
  *
- *            compress_model should contain the mode name.
- *                           Currently only "zlib" and "lz4" are supported.
- *                           Setting to "none" will disable compression.
+ *            compress_model contains the model name.
+ *                           See "compress_level" for the list of accepted values.
+ *                           The "none" value disables compression.
```

Comment:
'The "none" value disables compression' reads oddly to me. I think my original line is still easier for a native English speaker to parse. perhaps 'Setting the value to "none" disables compression' ?

Reply:
Thanks for the review, I certainly won't argue here. Do you feel like pushing a larger-scope fixup or should I add this single line change and leave the rest for later?

Reply:
I'm happy for you to just make that single-line change here.